File: compsci/programming/exersices/helsinki-moocs/java/ex26.java
import java.util.Scanner;

public class ex26 {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);

    int sum = 0;

    while(true) {
      System.out.print("Number: ");
      int user_input = Integer.parseInt(input.nextLine());
      sum += user_input;
      System.out.println("Sum now: "+ sum);

      if (user_input == 0) {
        System.out.println("Sum in the end: "+ sum);
        break;
      }
    }

  }// main entry func
}

File: compsci/programming/exersices/helsinki-moocs/c/ex26.c
#include <stdio.h>
#include <string.h>

int main(void) {
  int sum = 0;
  int user_number;

  // main program
  while (0 < 1) {
    printf("Type a number: ");
    scanf("%d", &user_number);
    sum += user_number;

    printf("\nThe sum now: %d\n", sum);

    if (user_number == 0) {
      break;
    }
  }

  return 0;
}
